harness/types/capabilities.test.ts

```diff
@@ -0,0 +1,34 @@
+import { Sdks } from './sdks'
+
+describe('SDKCapabilities Unit Tests', () => {
+    afterEach(() => {
+        jest.resetModules()
+    })
+
+    it('should load capabilities from env variables', () => {
+        process.env.SDK_CAPABILITIES = 'edgeDB,cloud,clientCustomData,sse'
+        process.env.SDKS_TO_TEST = 'NodeJS,Python'
+        const { SDKCapabilities } = jest.requireActual('../capabilities')
+        expect(SDKCapabilities).toEqual({
+            NodeJS: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+            Python: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+        })
+    })
+
+    it('should load capabilities from env variables as JSON', () => {
+        process.env.SDK_CAPABILITIES =
+            '["edgeDB","cloud","clientCustomData","sse"]'
+        process.env.SDKS_TO_TEST = '["NodeJS","Python"]'
+        const { SDKCapabilities } = jest.requireActual('../capabilities')
+        expect(SDKCapabilities).toEqual({
+            NodeJS: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+            Python: ['edgeDB', 'cloud', 'clientCustomData', 'sse'],
+        })
+    })
+
+    it('should use default capabilities if env variables are not set', () => {
+        process.env.SDK_CAPABILITIES = 'edgeDB,cloud,clientCustomData,sse'
```

Comment:
shouldnt this not be set then?

Reply:
The idea is for this to only be used when `SDKS_TO_TEST` is set. The idea is for the various SDKs to be able to set their capabilities when making breaking changes, but didn't want to turn both the existing `SDKS_TO_TEST` + `SDK_CAPABILITIES` into a single new JSON map you need to pass in. Seeing that we are only really going to use this for a single repo testing a new capability I thought this trade-off was fine.

Reply:
so this test is testing that basically `SDK_CAPABILITIES` isn't used when `SDKS_TO_TEST` isn't set.